src/main/java/eu/dissco/backend/service/SpecimenService.java

```diff
@@ -2,8 +2,7 @@
 
```

Comment:
maybe rename this to DigitalSpecimenService to be consistent? I don't love having 3 services that start with "Digital" though. So do what you want. this comment isn't very helpful.

Reply:
No I agree, this should be DigitalSpecimenService